modules/ROOT/pages/functions/graph.adoc

```diff
@@ -89,6 +89,22 @@ Properties for all graphs on the current composite database are returned.
 
 |===
 
+.Query
+[source, cypher, indent=0]
+----
+UNWIND graph.names() AS name
+WITH name, graph.propertiesByName(name) AS props
+WHERE any(tag IN props.tags WHERE tag IN ["A"])
```

Comment:
```suggestion
WHERE "A" IN props.tags
```
Since `tags` is a list we could just check if it contains `"A"` like this right?

Reply:
Right, this is equivalent